pkg/unidling/controller/unidling_controller.go

```diff
@@ -24,12 +24,14 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/pager"
 	"k8s.io/client-go/util/workqueue"
+	"k8s.io/component-base/metrics"
 	"k8s.io/klog/v2"
 
 	"github.com/openshift/api"
 	unidlingapi "github.com/openshift/api/unidling/v1alpha1"
 	appstypedclient "github.com/openshift/client-go/apps/clientset/versioned/typed/apps/v1"
 	"github.com/openshift/library-go/pkg/unidling/unidlingclient"
+	unidlingMetrics "github.com/openshift/openshift-controller-manager/pkg/unidling/metrics"
```

Comment:
Style nit - camelCase isn't typically used in imports.

```suggestion
	unidlingmetrics "github.com/openshift/openshift-controller-manager/pkg/unidling/metrics"
```